modules/understanding-upgrade-channels.adoc

```diff
@@ -69,7 +69,7 @@ You can use the `fast-{product-version}` channel to upgrade from a previous mino
 
 While the `fast-{product-version}` channel contains releases as soon
 as their errata are published, releases are added to the `stable-{product-version}` channel
-after a delay of several hours to a day. During this delay, data is collected from Red Hat SRE teams, Red Hat support services, and pre-production and production environments that participate in connected customer program about the stability of the release.
+after a delay of up to 48 hours for z-stream updates or several weeks for minor upgrades. During this delay, data is collected from Red Hat SRE teams, Red Hat support services, and pre-production and production environments that participate in connected customer program about the stability of the release.
```

Comment:
I'd be consistent with either "updates" and "upgrades," but I'm otherwise happy to merge this change.

Reply:
Thanks, fixed that. Overall the doc is all over the board 19 cases of 'update' and 25 of 'upgrade' but didn't seem right to clean all of those up.